modules/ROOT/pages/runtime-configure-gov.adoc

```diff
@@ -0,0 +1,87 @@
+wrapper.java.additional.<n>=-Danypoint.platform.client_secret=<environment client secret>
+wrapper.java.additional.<n>=-Danypoint.platform.base_uri=https://gov.anypoint.mulesoft.com
+wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.gov.anypoint.mulesoft.com
+----
+
+In the above example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.
+
+=== Command Line
+
+To configure Mule Runtime from the command line, use the following options when starting the Mule runtime:
+
+----
+./mule -M-Danypoint.platform.client_id=<environment client ID> -M-Danypoint.platform.client_secret=<environment client secret> -M-Danypoint.platform.base_uri=https://eu1.anypoint.mulesoft.com
+-M-Danypoint.platform.analytics_base_uri=https://analytics-ingest.gov.anypoint.mulesoft.com
+----
+
+=== CloudHub
+
+When deploying to CloudHub, you must include the following properties in the Properties tab. You must also ensure that you enable the autodiscovery feature.
+
+----
+anypoint.platform.client_id=<environment client ID>
+anypoint.platform.client_secret=<environment client secret>
+anypoint.platform.base_uri=https://gov.anypoint.mulesoft.com
+anypoint.platform.analytics_base_uri=https://analytics-ingest.gov.anypoint.mulesoft.com
+----
+
+== Configuring Mule Runtime 3.x
+
+=== wrapper.conf
+
+To configure `wrapper.conf` file, modify the following properties.
+
+----
+wrapper.java.additional.<n>=-Danypoint.platform.client_id=<org client ID>
+wrapper.java.additional.<n>=-Danypoint.platform.client_secret=<org client secret>
+wrapper.java.additional.<n>=-Danypoint.platform.platform_base_uri=https://gov.anypoint.mulesoft.com/apiplatform
+wrapper.java.additional.<n>=-Danypoint.platform.coreservice_base_uri=https://gov.anypoint.mulesoft.com/accounts
+wrapper.java.additional.<n>=-Danypoint.platform.contracts_base_uri=https://gov.anypoint.mulesoft.com/apigateway/ccs
+wrapper.java.additional.<n>=-Danypoint.platform.analytics_base_uri=https://analytics-ingest.gov.anypoint.mulesoft.com
+----
+
+In the above example, replace `<n>` with an integer value that is unique within the `wrapper.conf` file.
+
+=== Command Line
+
+To configure Mule Runtime from the command line, use the following options when starting the Mule runtime:
+
+----
+./mule -M-Danypoint.platform.client_id=<org client ID> -M-Danypoint.platform.client_secret=<org client secret> -M-Danypoint.platform.platform_base_uri=https://eu1.anypoint.mulesoft.com/apiplatform -M-Danypoint.platform.coreservice_base_uri=https://eu1.anypoint.mulesoft.com/accounts -M-Danypoint.platform.contracts_base_uri=https://eu1.anypoint.mulesoft.com/apigateway/ccs -M-Danypoint.platform.analytics_base_uri=https://analytics-ingest.eu1.anypoint.mulesoft.com
+----
+
+=== CloudHub
+
+When deploying to CloudHub, you must add the following in the Properties tab. You must also ensure that you enable the autodiscovery feature.
```

Comment:
Add the following details or information? in the Properties tab: Make sure that you enable the autodiscovery feature.

Reply:
Reworded this - hoping that it works with the placement of the Properties tab. Please let me know if think it's ambiguous now. Thanks!